snippets/visualbasic/how-to/conversions/Program.vb

```diff
@@ -11,7 +11,7 @@ Module Program
     Sub FirstExample()
         ' <Snippet1>
         Dim MyString As String = "Jan 1, 2009"
-        Dim MyDateTime As DateTime = DateTime.Parse(MyString)
+        Dim MyDateTime As Date = Date.Parse(MyString)
         Console.WriteLine(MyDateTime)
```

Comment:
I think it should stay `DateTime` as it was. This code is referenced in the [How to: convert strings to DateTime](https://docs.microsoft.com/en-us/dotnet/standard/base-types/parsing-datetime) article.
The article speaks about `DateTime`, not `Date`. The edit that may be done here is to use camelCase for local variables.

Reply:
I can put it back but VS 2019 has a code fix to replace DateTime with Date, so the article should be updated not the code. This code will produce the same results and is now the preferred (by VS 2019) way. Let me know what I should do.

Reply:
The article is for both C#, and VB.NET.
As far as I know, In VB.NET, `Date` is an alias for `DateTime`, but in C#, there is no `Date`.
so I think using `DateTime` is more suitable to be compatible with both languages.

Reply:
I restored it, VB/VS prefers Date but since there is one article let's go with that. Somewhere in the article it would be good to cover this so people are not surprised when the IDE recommends  changing it.

Reply:
Done